scanner/vite.config.ts

```diff
@@ -25,4 +25,11 @@ export default defineConfig({
       ],
     },
   },
+  css: {
+    preprocessorOptions: {
+      scss: {
+        additionalData: `@import './src/styles/variables.scss';`,
+      },
+    },
+  },
```

Comment:
No. Use explicit imports.

Reply:
If there isn't an alias for `$styles` yet, make one pointing to `src/styles`

Reply:
This is not an explicit import, no matter what path you use. I meant to put the `@import` explictly where you need it, i.e. in the components that use the variables.